installers/install.ps1

```diff
@@ -185,10 +185,14 @@ function warningIfadmin() {
 
```

Comment:
We don't need to modify these installers on the release branch, as by the time this lands it'd be the ones on the current beta branch that'd end up being used.

That said, we need to revisit our legacy-installers to fully address this. I'll file a story for that.

Reply:
https://www.pivotaltracker.com/story/show/179305221

Reply:
Yes, you are right.  The installers are by now almost the same though, only that the `legacy-install` scripts always require a version.  So, it shouldn't be too complicated...